From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Wed, 8 Mar 2023 09:51:20 +0100
Subject: [PATCH] Release 6.3/customers/lxa/lxatac/20230308-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d7bd0eb9b346..6228956adebc 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 3
 SUBLEVEL = 0
-EXTRAVERSION = -rc1
+EXTRAVERSION = -rc1-20230308-1
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
