From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Fri, 1 Mar 2024 11:23:27 +0100
Subject: [PATCH] Release 6.8/customers/lxa/lxatac/20240301-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9869f57c3fb3..2217e50dacda 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 8
 SUBLEVEL = 0
-EXTRAVERSION = -rc1
+EXTRAVERSION = -rc1-20240301-1
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
