From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Thu, 30 Mar 2023 16:37:52 +0200
Subject: [PATCH] Release 6.3/customers/lxa/lxatac/20230330-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index da2586d4c728..42a330454c79 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 3
 SUBLEVEL = 0
-EXTRAVERSION = -rc4
+EXTRAVERSION = -rc4-20230330-1
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
